Dockerfile

```diff
@@ -238,6 +239,9 @@ RUN cargo install --version=${deadlinks_version} cargo-deadlinks
 # cf. https://github.com/rust-fuzz/cargo-fuzz/pull/277
 RUN cargo install --git https://github.com/rust-fuzz/cargo-fuzz/ --rev 8c964bf183c93cd49ad655eb2f3faecf543d0012
 
+# Install cargo-binutils.
+RUN cargo install --git https://github.com/rust-embedded/cargo-binutils --rev 36102732f7535b4730f7cd66c670ebe3959994ef
```

Comment:
It looks like there are prebuilt release binaries for this: https://github.com/rust-embedded/cargo-binutils/releases/tag/v0.3.6

I think it is better to depend on those directly if possible, since eventually they will be part of the provenance generation, and, paradoxically, being able to say that we use a specific prebuilt binary with a given hash is a stronger story than saying we built it ourselves with our own Rust toolchain.